style.css

```diff
@@ -0,0 +1,206 @@
+.navigation > div + div:active{
+  background-color: #CCC;
+}
+
+.navigation > div + div + div + div + div + div + div + div + div + div + div {
+  z-index: -1;
+}
+.navigation > div + div + div + div + div + div + div + div + div + div + div:hover {
+  background-color: white;
+}
+.navigation > div + div + div + div + div + div + div + div + div + div + div:active {
+  background-color: white;
+}
+
+.main .compose-grid > .compose-post  .tweet-button:hover,
+.navigation .tweet-button:hover {
+  background-color: #2D92EB;
+
+}
+.main .compose-grid > .compose-post  .tweet-button:active,
+.navigation .tweet-button:active {
+  background-color: #4870AB;
+}
+
+.navigation img {
+  margin-right: 9px;
+  width: 30px;
+}
+
+.navigation > div {
+  font-weight: bold;
+}
+
+.navigation > div > div {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+  margin: 3px;
+  background-color: #4BA0EB;
+  border-radius: 4px;
+
+}
+
+.navigation > .tweet-button {
+  margin-top: 20px;
+  justify-content: center;
+  align-items: center;
+  border-radius: 40px;
+  width: 100%;
+  height: 100%;
+  justify-self: center; 
+  align-self: center;
+  background-color: #4BA0EB;
+  color: white;
+}
+
+/* #home { */
+/*   display: none; */
+/* } */
+
+.main {
+  display: grid;
+  grid-template-rows: 50px 150px 1fr;
+  min-height: 100vh;
+}
+
+.main {
+  background-color: #DDD;
+}
+
+.main > div {
+  margin: 2px;
+  background-color: white;
+}
+
+.main .compose-grid {
+  display: grid;
+  grid-template-columns: 90px 1fr;
+}
+
+.main > .compose-grid > .profile-pic {
+  height: 70px;
+  width: 70px;
+  border-radius: 50%;
+}
+
+.main .compose-grid > .compose-post {
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  border: none;
+}
+
+.main .compose-grid > .compose-post > .post-attachments {
+  display: grid;
+  grid-template-columns: 40px 40px 40px 40px 40px 1fr 200px;
+  column-gap: 20px;
+  align-items: center;
+}
+
+.main .compose-grid > .compose-post  .tweet-button {
+  display: grid;
+  border-radius: 40px;
+  width: 90%;
+  height: 90%;
+  background-color: #4BA0EB;
+  color: white;
+  align-items: center;
+  justify-items: center;
+}
+
+.main .compose-grid > .compose-post > .post-attachments > div > img {
+  align-self: center;
+  height: 43px;
+  max-width: 43px;
+}
+
+.main .feed {
+  display: grid;
+  grid-template-rows: repeat(auto-fit, 200px);
+  background-color: #DDD;
+}
+
+.main > .feed > .feed-post {
+  display: grid;
+  grid-template-columns: 90px 1fr;
+  margin-top: 2px;
+  background-color: #FFF;
+}
+
+.main > .feed > .feed-post > .feed-profile-pic {
+  height: 70px;
+  width: 70px;
+  border-radius: 50%;
+}
+
+.main > .feed > .feed-post > .feed-body {
+  display: grid;
+  grid-template-rows: 50px minmax(70px, 1fr) 60px;
+}
+
+.happening {
+  display: grid;
+  grid-template-rows: repeat(auto-fit, 60px);
+}
+
+.happening > div {
+  margin: 4px;
+  border: 3px solid black;
+  border-radius: 7px;
+}
```

Comment:
I think it looks nice overall but the code of the html and CSS is strikingly short. At the end I ask myself if the twitter page is complete? Or is there still more to add/adjust/finish?

The length of the code makes it hard to believe that you have finished, and if you have finished everything then hats off to you very well done!

Reply:
Hi Marc, thought the structure is all here, there's definitely a bit more to add content-wise.

Reply:
Nice work mate, very clean and easy to read! 💯